Admin/AbstractAdmin.php

```diff
@@ -2172,12 +2172,14 @@ public function getTranslationDomain()
      *
      * @deprecated since 3.9, to be removed with 4.0
      */
-    public function setTranslator(TranslatorInterface $translator)
+    public function setTranslator(TranslatorInterface $translator, $deprecation = true)
```

Comment:
This trick is nice, but…neither the method nor the class are final, so [this is a BC-break](https://3v4l.org/dVLLZ). I just thought of another solution 💡  Why not use `func_get_args()` ?